lib/Doctrine/ORM/EntityManager.php

```diff
@@ -156,6 +164,7 @@ protected function __construct(Connection $conn, Configuration $config, EventMan
         $this->metadataFactory->setEntityManager($this);
         $this->metadataFactory->setCacheDriver($this->config->getMetadataCacheImpl());
 
+        $this->persisterFactory  = new PersisterFactory($this);
```

Comment:
> EntityGraph which would drastically simplify UoW by being part of another class ChangeSet that tracks Entity states.

I'm already designing that stuff, but it's out of scope for this PR.